### DISCUSSION TITLE
IS there any extension in ZED for API testing clients. Like postman

### DISCUSSION THREAD

**Discussion by limon513** [zed-industries/zed#14426] [Feature Requests]
IS there any extension in ZED for API testing clients. Like postman , thanks in advance

**Comment by hashims** [comment]
no there isn't atm, and not in roadmap also

**Reply by semiautomatix** [reply]
Boo!

**Reply by semiautomatix** [reply]
:wink: 

I've only just discovered Postman in VSCode and it makes a nice addition. No reason there couldn't be an extension.

**Reply by ivanbarlog** [reply]
Or maybe something like [Rest Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) would be nice. Is it hard to port VSCode extensions to Zed?

**Reply by jrvidotti** [reply]
REST client is awesome. It's my first choice for VS Code.

**Reply by honcur** [reply]
REST client is nice

**Comment by zboinek** [comment]
What do you think about storing request definition per project as files and run them through command pallete with use of curl or something?

**Comment by r-moore** [comment]
looks like someone is having a go here https://github.com/selfhostcloud/zed-httpclient

**Comment by dev-kvt** [comment]
it would be fun if they added one
